android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -320,6 +325,30 @@ class DVCClient private constructor(
         }
     }
 
+    private fun refetchConfig(callback: DVCCallback<Map<String, Variable<Any>>>? = null) {
```

Comment:
should we have any tests for this method?

Reply:
we only have tests for public methods so i figured we'd test it by testing the methods that call it (i guess by mocking messages from the sse connection and expecting the outcome to be a new config?)

Reply:
is there no way to test private methods? not that familiar with testing on mobile

Reply:
okay i finally figured it out and added a test, you just need to get a reference to the method itself and set it accessible